### PR TITLE
RIPE RPKI archive format changed (.xz) #14

### PR DESCRIPTION
**Pull request by MAVRICK-1**
Updated the URLS OF RPKI_ARCHIVE_URLS and uncompressed it .

solving the issue of #14 

```cmd

PS C:\Users\RISHI\Desktop\route-origin-validator\rov> rov  8.8.8.0/24 15169 --rpki_archive 2024/01/12
Downloading: https://ftp.ripe.net/rpki/afrinic.tal/2024/01/12/roas.csv.xz
Downloading: https://ftp.ripe.net/rpki/apnic.tal/2024/01/12/roas.csv.xz
Downloading: https://ftp.ripe.net/rpki/arin.tal/2024/01/12/roas.csv.xz
Downloading: https://ftp.ripe.net/rpki/lacnic.tal/2024/01/12/roas.csv.xz
Downloading: https://ftp.ripe.net/rpki/ripencc.tal/2024/01/12/roas.csv.xz
Loading: C:\Users\RISHI\AppData\Local\IHR\rov\Cache/db/delegated\nro-extended-stats
Loading: C:\Users\RISHI\AppData\Local\IHR\rov\Cache/db/irr\+lacnic.db.gz
Loading: C:\Users\RISHI\AppData\Local\IHR\rov\Cache/db/irr\lacnic.db.gz
Loading: C:\Users\RISHI\AppData\Local\IHR\rov\Cache/db/rpki//2024/01/12\afrinic.csv
Loading: C:\Users\RISHI\AppData\Local\IHR\rov\Cache/db/rpki//2024/01/12\apnic.csv
Loading: C:\Users\RISHI\AppData\Local\IHR\rov\Cache/db/rpki//2024/01/12\arin.csv
Loading: C:\Users\RISHI\AppData\Local\IHR\rov\Cache/db/rpki//2024/01/12\lacnic.csv
Loading: C:\Users\RISHI\AppData\Local\IHR\rov\Cache/db/rpki//2024/01/12\ripencc.csv
{
    "query": {
        "prefix": "8.8.8.0/24",
        "asn": 15169
    },
    "irr": {
        "status": "NotFound"
    },
    "rpki": {
        "status": "Valid",
        "prefix": "8.8.8.0/24",
        "maxLength": 24,
        "ta": "unknown",
        "startTime": "2023-12-29 17:14:18",
        "endTime": "2024-03-28 16:14:18",
        "uri": "rsync://rpki.arin.net/repository/arin-rpki-ta/5e4a23ea-e80a-403e-b08c-2171da2157d3/f60c9f32-a87c-4339-a2f3-6299a3b02e29/5e9328a9-e1d2-45d8-bdb5-eefe152994f9/d74ad38d-11d3-3c79-a22a-29626dd55bea.roa"
    },
    "delegated": {
        "prefix": {
            "status": "assigned",
            "prefix": "8.8.8.0/24",
            "date": "20231228",
            "registry": "arin",
            "country": "US"
        },
        "asn": {
            "status": "assigned",
            "registry": "arin"
        }
    }
}

```

I have only shown the output for  RPKI_ARCHIVE_URLS  didn't included the other URLS in the output

@romain-fontugne review my PR :-)

### CODE REVIEW

**Comment by MAVRICK-1** [comment]
@romain-fontugne can your pls review this PR when u got time 🙏☺️

**Comment by MAVRICK-1** [comment]
@romain-fontugne can your pls review this PR when u got time 🙏☺️

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 appdirs
 py-radix
 portion
+lzma
```
Comment (romain-fontugne) [inline requirements.txt:4]: I guess this should be pylzma?

File: setup.py [hunk 2]
```diff
@@ -18,6 +18,7 @@
         'appdirs',
         'py-radix',
         'portion',
+        'lzma'
```
Comment (romain-fontugne) [inline setup.py:21]: I guess this should also be pylzma

Comment (romain-fontugne) [reply setup.py:21]: Let's also bump the version to 0.5.0

**Review by romain-fontugne — CHANGES_REQUESTED** [review]
There is some problems with the dependencies and to install the app, otherwise that looks as expected, thanks!

Please see my comments

**Comment by MAVRICK-1** [comment]
@romain-fontugne  sir my fault, the `lzma` module is part of the standard library .
Rectifying it

**Comment by MAVRICK-1** [comment]
@romain-fontugne what you think about #19 ?

File: requirements.txt (hunk 1 above)
Comment (MAVRICK-1) [reply requirements.txt]: > I guess this should be pylzma?
 
I rectified it. The lzma module is part of the standard library ,So dependency and installation not needed.

File: setup.py (hunk 2 above)
Comment (MAVRICK-1) [reply setup.py]: updated the bump version.

**Review by romain-fontugne — APPROVED** [review]
Looks good to me thanks!

**Pull request merged**